### ISSUE TITLE
Upgrade to v2 API

### ISSUE THREAD

**Issue by davidus05** [dev-labs-bg/sportify#30]
The API-provider https://www.football-data.org/ just changed to v2. So the old v1 version will be removed in the future.

A upgrade of the script would be great. Thanks!

**Comment by aminvakil** [comment]
Does it work until the end of WC2018?

**Comment by davidus05** [comment]
Yes, I think so. -> https://www.football-data.org/documentation/quickstart

_"The old v1 API will remain available for the foreseeable future, but I do recommend you migrate to the new v2 API ASAP. The v1 API documentation can still be accessed."_

**Comment by cmihaylov** [comment]
World Cup 2018 should be fine.
It doesn't look like v1 will soon stop working, but I will look into the changes introduced in v2 and see what we need to change in the logic to make it work with it.

**Comment by aminvakil** [comment]
Any progress on this?
As UCL 18/19 will continue to next year it should be changed to make it compatible with v2.
Here is changelog:
https://www.football-data.org/documentation/api#changelog
I will try to change it to make it work with v2.

**Comment by aminvakil** [comment]
I've made it compatible with v2, but I can't open a pull request to push my changes to repository, What should I do? @cmihaylov @davidus05

**Comment by davidus05** [comment]
Great work, thanks!
I saw your pull already here: https://github.com/dev-labs-bg/sportify/pull/34
I will have to modify the files manually I think - I see your changes there. Maybe @cmihaylov will remove this restriction.

**Comment by aminvakil** [comment]
I'm idiot in git. Please fix it yourself.
Thanks. @davidus05

**Comment by cmihaylov** [comment]
@aminvakil great work on the api v2 changes. I will review your pull request later today and merge it.

**Comment by aminvakil** [comment]
So far I checked it on importing UCL matches only, maybe we should wait until the end of day and see what happens after games today and will we face any problem or not. @cmihaylov

**Comment by cmihaylov** [comment]
OK, cool. I'll try to validate it myself too if all works as expected.

**Comment by cmihaylov** [comment]
I did some minor fixes to the pull request and merged it.
